rubygem-foreman_openscap/rubygem-foreman_openscap.spec

```diff
@@ -15,20 +15,19 @@
 %global foreman_bundlerd_dir %{foreman_dir}/bundler.d
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.4.3
-Release: 2%{?foremandist}%{?dist}
+Version: 0.5.0
+Release: 1%{?foremandist}%{?dist}
 Summary: Foreman plug-in for displaying OpenSCAP audit reports
 Group: Applications/System
 License: GPLv3
 URL: https://github.com/OpenSCAP/foreman_openscap
 Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
 
-Requires: foreman >= 1.8.0
+Requires: foreman >= 1.11.0
 
 Requires: scap-security-guide
 Requires: %{?scl_prefix}rubygem(deface) < 2.0
-Requires: %{?scl_prefix}rubygem(scaptimony) >= 0.3.2
-Requires: %{?scl_prefix}rubygem(scaptimony) < 0.4.0
+Obsoletes: %{?scl_prefix}rubygem(scaptimony) >= 0.3.2
```

Comment:
It'd be `< 0.3.2-3` in this case, since you're aiming to obsolete all previously shipped versions, not future/newer versions.

(https://fedoraproject.org/wiki/Upgrade_paths_%E2%80%94_renaming_or_splitting_packages)